content/doc/developer/internationalization/internationalization-and-localization.adoc

```diff
@@ -0,0 +1,320 @@
+---
+title: Internationalization and Localization
+layout: developersection
+---
+
+https://www.jenkins.io/doc/book/using/using-local-language/[How
+to view Jenkins in your language].
+
+'''''
+
+== Translations to Specific languages
+
+* https://wiki.jenkins.io/JENKINS/65670018[Chinese
+(Traditional) Translation]
+* https://wiki.jenkins.io/JENKINS/135469573[Chinese
+(Simplified) Translation]
+* https://wiki.jenkins.io/JENKINS/French-Localization[French
+Localization]
+* https://wiki.jenkins.io/display/JA/Japanese-Localization[Japanese
+Localization]
+* https://wiki.jenkins.io/JENKINS/Polish-Localization[Polish
+Localization]
+* https://wiki.jenkins.io/JENKINS/Spanish-Translation[Spanish
+Translation]
+* https://wiki.jenkins.io/JENKINS/Brazilian-Portuguese-Translation[Brazilian
+Portuguese Translation]
+
```

Comment:
These pages are badly outdated and cannot be easily modified.  If we need that information, then we need to insert it into pages in this repository rather than linking to the pages on the read-only Jenkins wiki.
```suggestion
```